nemo/utils/model_utils.py

```diff
@@ -338,13 +338,28 @@ def resolve_validation_dataloaders(model: 'ModelPT'):
             # using the name of each of the nested dataset
             model._validation_names = [ds.name for ds in ds_values]
         else:
-            model._validation_names = [parse_dataset_as_name(ds) for ds in ds_values]
+            ds_names = cfg.validation_ds.get('name', [])
```

Comment:
👏🏻. Thank you! I hope this solves issues with validation dl when restarting from previous runs with different number of validation dls